Tools-Override/CloudTest.targets

```diff
@@ -493,10 +476,9 @@
         <Build Condition="'$(IsOfficial)'=='true'">$(OfficialBuildId)</Build>
         <Type>$(HelixJobType)</Type>
         <Source>$(HelixSource)</Source>
-        <Properties>{ &quot;architecture&quot; : &quot;$(Platform)&quot;, &quot;configuration&quot;: &quot;$(ConfigurationGroup)&quot;, &quot;operatingSystem&quot; : &quot;$(TargetQueue)&quot; }</Properties>
+        <Properties>{ &quot;architecture&quot; : &quot;$(ArchGroup)&quot;, &quot;configuration&quot;: &quot;$(ConfigurationGroup)&quot;, &quot;operatingSystem&quot; : &quot;$(TargetQueue)&quot; }</Properties>
```

Comment:
While I'm OK with this for now, as you would guess I'm going to say this is very much a corefx'ism so if this is interesting it should be an extension point in something that ultimately lives in corefx.